pkg/loader/executor.go

```diff
@@ -146,12 +150,11 @@ func (e *executor) bulkReplace(inserts []*DML) error {
 	}
 
 	info := inserts[0].info
-	dbName := inserts[0].Database
-	tableName := inserts[0].Table
 
-	builder := new(strings.Builder)
+	var builder strings.Builder
 
-	fmt.Fprintf(builder, "REPLACE INTO %s(%s) VALUES ", quoteSchema(dbName, tableName), buildColumnList(info.columns))
+	cols := buildColumnList(info.columns)
+	builder.WriteString(fmt.Sprintf("REPLACE INTO %s(%s) VALUES ", inserts[0].TableName(), cols))
```

Comment:
Same comment as https://github.com/pingcap/tidb-binlog/pull/579#discussion_r281110602. If you think that's not a problem then LGTM.

Reply:
Updated, PTAL.